CONTRIBUTORS.txt

```diff
@@ -12,6 +12,7 @@ Kanikaa Murarka (gh: kanikaa1234)
 Aditya Chawla (gh: chawlaaditya8)
 Vipin Khushu (gh: vipinkhushu)
 Saurabh Badhwar (gh: h4xr)
+Mayank Badola (gh: mbad0la)
```

Comment:
This change should not be part of this pull request.

Reply:
Also, please update your branch with latest master changes.